backend/src/utils/transformer/transformers/mopidy/tracklist/index.js

```diff
@@ -1,8 +1,17 @@
 import TransformTrack from 'utils/transformer/transformers/mopidy/track'
+import { findTracks } from 'utils/track'
 
 const Tracklist = (json) => {
-  return json.map(data => {
-    return TransformTrack(data)
+  return new Promise((resolve) => {
+    const trackUris = json.map(data => data.uri)
+    findTracks(trackUris).then(tracks => {
```

Comment:
Doing a find on all the playlist IDs in one go rather than inside a loop finding each one